integrationExamples/gpt/idImportLibrary_example.html

```diff
@@ -0,0 +1,351 @@
+                  name: "sharedid",
+                  expires: 28
+                }
+              } ,  {
+                name: "liveIntentId",
+                params: {
+                  publisherId: "9896876"
+                },
+                storage: {
+                  type: "cookie",
+                  name: "_li_pbid",
+                  expires: 28
+                }
+              },
+              {
+                name: "zeotapIdPlus"
+              },
+              {
+                name: 'haloId',
+                storage: {
+                  type: "cookie",
+                  name: "haloId",
+                  expires: 28
+                }
+              }, {
+                name: "merkleId",
+                params: {
+                  ptk: '12345678-aaaa-bbbb-cccc-123456789abc', //Set your real merkle partner key here
+                  pubid: 'EXAMPLE' //Set your real merkle publisher id here
+                },
+                storage: {
+                  type: "html5",
+                  name: "merkleId",
+                  expires: 30
+                },
+
+              },{
+                name: "unifiedId",
+                params: {
+                  partner: "prebid",
+                  url: "http://match.adsrvr.org/track/rid?ttd_pid=prebid&fmt=json"
+                },
+                storage: {
+                  type: "html5",
+                  name: "unifiedid",
+                  expires: 30
+                },
+              },{
+                name: "parrableId",
+                params: {
+                  // change to Parrable Partner Client ID(s) you received from the Parrable Partners you are using
+                  partner: '30182847-e426-4ff9-b2b5-9ca1324ea09b'
+                }
+              },{
+                name: "criteo"
+              }
+            ],
+            syncDelay: 5000,
+            auctionDelay: 1000
+          }
+        });
+        pbjs.addAdUnits(adUnits);
+        pbjs.requestBids({
+          bidsBackHandler: sendAdserverRequest
+        });
+      });
+
+      function sendAdserverRequest() {
+        if (pbjs.adserverRequestSent) return;
+        pbjs.adserverRequestSent = true;
+        googletag.cmd.push(function() {
+          pbjs.que.push(function() {
+            pbjs.setTargetingForGPTAsync();
+            googletag.pubads().refresh();
+          });
+        });
+      }
+      var FAILSAFE_TIMEOUT = 2000;
+
+      setTimeout(function() {
+        sendAdserverRequest();
+      }, FAILSAFE_TIMEOUT);
+    </script>
+
+    <script>
+      googletag.cmd.push(function() {
+        googletag.defineSlot('/112115922/FL_PB_MedRect', [[300, 250],[300,600]], 'test-div').addService(googletag.pubads());
+        googletag.pubads().enableSingleRequest();
+        googletag.enableServices();
+      });
+      pbjs.que.push(function() {
+        pbjs.setConfig({
+          idImportLibrary:{
+            url: 'http://localhost:8090/sync'
```

Comment:
I assume it's fine that this doesn't actually work, since it's just a demo, right?

Reply:
Yes its fine. The url is validated so just added a local host server there.

Reply:
ok thanks